source/common/res/features/shared/main.js

```diff
@@ -281,7 +281,7 @@ ynabToolKit.shared = (function () {
       }
     },
 
-    showModal(title, message, actionType) {
+    showModal(logo, title, version, message, actionType) {
```

Comment:
@danieldbird this method was meant to be pretty generic where you would send a title and a message and it would render a ynab-like modal for you. adding all of the version-specific things will break that pattern.

we should probably just create another function that will show a modal rather than updating this function.

Reply:
👍  ok